packages/x-components/static-docs/build-search-ui/web-archetype-integration-guide.md

```diff
@@ -209,31 +185,58 @@ events.
+
+* **Initialization options** supported in [snippet configuration](#snippet-configuration)
+* **[X&nbsp;event&nbsp;types](#interface-x-events-types)** available to subscribe to via callbacks
+* **[Functions supported by the X&nbsp;API object](#x-api)** to initialize Interface&nbsp;X
+
+#### Snippet configuration
+
+The [snippet configuration](https://github.com/empathyco/x-archetype/blob/main/public/snippet-script.js) allows you to configure multiple initialization options for the Interface&nbsp;X project such as language, currency, and shopper's personal data consent. The snippet configuration supports the following configuration options:
 
-The XEventsTypes are those that can be triggered from different modules inside the X Components, but
-have in mind that every module has its own sort of components. So have a look to DeviceXEvents,
-EmpathizeXEvents, SearchBoxXEvents, etc.
+| Name                    | Type                                                                                 |  Description  |
+| ----------------------- | ------------------------------------------------------------------------------------ | ------ | 
+| `instance`                | `string`                                                                             | _Required._ ID of the API client instance. It's provided by Empathy. |
+| `env`                     | `'live'` &#124; `'staging'`              | API environment to use. You can use the Interface&nbsp;X  production version with the staging API, and vice versa.  |
+| `scope`                   | `string`    | Context where the search interface is executed, i.e. `mobile`, `mobile-app`, `tablet`, `desktop`.  |
+| `lang`                    | `string`                                                                             | _Required._ Language to use. By default, it's used for both the frontend and the API requests.   |
+| `searchLang`              | `string`     | Language to use for the API requests **only**.  |
+| [`consent`](#consent)                | `boolean`    |_Required._ Determines whether the shopper has accepted the use of cookies so that the `sessionId` is sent to the Empathy's Search and Tagging APIs or not.  |
+| `documentDirection`       | `'ltr'` &#124; `'rtl'`    | Writing direction script that the X Components should, i.e. left-to-right or right-to-left.   |
+| `currency`                | `string`   |_Required._ Currency identifier to configure how prices are displayed.  |
+| [`callbacks`](#callbacks) | `Record<XEventName, (payload: XEventPayload<Event>, metadata: WireMetadata) => void` |    Callback record where the _key_ is the event to listen and the _value_ is the callback to be executed whenever the event is emitted. E.g. to listen to the `UserAcceptedAQuery` event: `{ UserAcceptedAQuery({ eventPayload }) { console.log('UserAcceptedAQuery', eventPayload); }  |
```

Comment:
Are callback and consent links working? I can't find their target, but they seem to link to a random place in the document.

Reply:
There was a callbacks section but Luisa got rid of it, but we are going to restore it, righ @mlcorton ? :)

Reply:
My fault! Sorry, I misunderstood the info provided! I'll revert the initial structure